daos-stack/daos
# DAOS-9515 test: Add container query with property test

Verify container properties are set during container query
over pydaos API.

Test Steps:
1. Create a container with some properties related to checksum, type, etc.
configured.
2. Call container query. it returns container info such as UUID, snapshots, etc.,
but not properties. We verify the property by passing in an empty data structure
as pass by reference.
3. Verify the property data structure passed in during step 2.

Added daos_property parameter in query() and fixed some typo.

Tag - Test
query_properties - container/query_properties.py
basic_checksum_object - checksum/csum_basic.py
containercreate - container/create.py
contdelete - contaienr/delete.py
cont_create_async - container/async.py
closehandle - container/open_close.py
snapshots - container/snapshot.py
dkey_akey_enum_punch - telemetry/dkey_akey_enum_punch.py

Skip-unit-tests: true
Test-tag: query_properties basic_checksum_object containercreate contdelete cont_create_async closehandle snapshots dkey_akey_enum_punch

Signed-off-by: Makito Kano <makito.kano@intel.com>
